examples/huggingface/pytorch/text-generation/quantization/requirements.txt

```diff
@@ -10,4 +10,4 @@ intel_extension_for_pytorch
 git+https://github.com/intel/neural-compressor.git
 git+https://github.com/huggingface/optimum.git@deda7e3be202e2a21a7d53dd5c284e47f9c646b7
 git+https://github.com/huggingface/optimum-intel.git@74912c0caa7dfb8979f59cbc94b57f5d6a448c30
-git+https://github.com/EleutherAI/lm-evaluation-harness.git@83dbfbf6070324f3e5872f63e49d49ff7ef4c9b3
```

Comment:
what's the reason to upgrade the commit?

Reply:
some changes (such as  lambada.py acc=0) have improved in the lm-eval.
support chatglm3.
the commit id we used is 6 months ago, lm-eval also have some updates to support int4/int8 woq and gptq.
https://github.com/EleutherAI/lm-evaluation-harness/tree/master